### PR TITLE
Jakarta EE 10 TCK results

### PR DESCRIPTION
**Pull request by arjantijms**
Signed-off-by: Arjan Tijms <arjan.tijms@gmail.com>

### CODE REVIEW

File: docs/website/src/main/resources/certifications/jakarta-platform/10/TCK-Results-7.0.md [hunk 1]
```diff
@@ -0,0 +1,772 @@
...
+  
+- Any Additional Specification Certification Requirements: <br/>
+  Jakarta Concurrency 3.0 TCK <br/>
+  Jakarta JSON Processing 2.1 TCK <br/>
+  Jakarta JSON Binding 3.0 TCK <br/>
+  Jakarta RESTful Web Services 3.1 TCK <br/>
+  Jakarta Security 3.0 TCK <br/>
+  Jakarta Authentication 3.0 TCK <br/>
+  Jakarta Faces 4.0 TCK <br/>
+  Jakarta Contexts and Dependency Injection 4.0 TCK <br/>
```
Comment (scottmarlow) [inline docs/website/src/main/resources/certifications/jakarta-platform/10/TCK-Results-7.0.md:33]: I'm trying to better understand the requirement for also passing the CDI 4.0 Language Model TCK.  Related to that is https://github.com/jakartaee/cdi-tck/issues/391

**Review by dmatej — APPROVED** [review]
I don't see any issue, however I could still miss something

File: docs/website/src/main/resources/certifications/jakarta-platform/10/TCK-Results-7.0.md [hunk 2]
```diff
@@ -0,0 +1,772 @@
...
+Fail
+(diff)
+Skip
+(diff)
+Pass
+(diff)
+Total
+(diff)
+org.atinject.tck.auto   6 ms    0       0       50      50  
+```
```
Comment (pzygielo) [inline docs/website/src/main/resources/certifications/jakarta-platform/10/TCK-Results-7.0.md:751]: This renders kind of unfriendly...

File: docs/website/src/main/resources/certifications/jakarta-platform/10/TCK-Results-7.0.md (hunk 1 above)
Comment (scottmarlow) [reply docs/website/src/main/resources/certifications/jakarta-platform/10/TCK-Results-7.0.md:33]: All clear now, passing CDI should include running the [CDI Language Model TCK](https://github.com/jakartaee/cdi-tck/blob/master/doc/reference/src/main/asciidoc/executing.asciidoc#executing-the-lang-model-test-suite).

File: docs/website/src/main/resources/certifications/jakarta-platform/10/TCK-Results-7.0.md [hunk 3]
```diff
@@ -0,0 +1,772 @@
...
+   [runcts] OUT => [javatest.batch] Number of Tests Passed      = 211
+   [runcts] OUT => [javatest.batch] Number of Tests Failed      = 0
+   [runcts] OUT => [javatest.batch] Number of Tests with Errors = 0
+   [runcts] OUT => [javatest.batch] ********************************************************************************
+
+Stage Name: ejb30/lite/singleton
+   [runcts] OUT => [javatest.batch] ********************************************************************************
+   [runcts] OUT => [javatest.batch] Completed running 230 tests.
+   [runcts] OUT => [javatest.batch] Number of Tests Passed      = 230
+   [runcts] OUT => [javatest.batch] Number of Tests Failed      = 10
```
Comment (hs536) [inline docs/website/src/main/resources/certifications/jakarta-platform/10/TCK-Results-7.0.md:196]: @arjantijms Here is a little strange. Is this a TCK error?

Comment (arjantijms) [reply docs/website/src/main/resources/certifications/jakarta-platform/10/TCK-Results-7.0.md:196]: Thanks for checking! Probably a copy paste error in some way by me.

I copied everything from https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck/job/master/1898/artifact/ you have to download each of the 150MB files, then hunt in the logs for the right snippet. It's very error prone and easy to make mistakes :(

**Pull request merged**